scss/scaffold.scss

```diff
@@ -129,6 +129,7 @@ vbox > splitter {
 
 #left-tabbox {
 	margin: 5px;
+	min-width: 500px;
```

Comment:
Oh, for me the 500px is approximately the content elements' non-overflow width on Windows. We can remove that, but without a minimal width, the left pane can be resized to smaller than its content...

Reply:
To prevent this: 
<img width="1008" alt="image" src="https://github.com/zotero/zotero/assets/33902321/77e2579e-2a6d-49ac-a3b6-f5e6a32973d5">

> the checkboxes have to wrap

That's another issue of the checkbox's container, the fx115 layout issue that we are already familiar with. Will fix

Reply:
Pushed a fix

Reply:
This PR was initially based on the current main, but since we want to merge fx115 ASAP, we decided to rebase this PR to fx115, which brings tons of fx115 fixes to make it work on fx115. Some parts might not be very related to the original PR.

The scaffold commits can be definitely picked into another separate PR if we want. Anyway we will need to go through these of commits and squash or rename them.